pkg/subscription/volsync.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2020 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package subscription
+
+import (
+	operatorsv1 "github.com/stolostron/multiclusterhub-operator/api/v1"
+	"github.com/stolostron/multiclusterhub-operator/pkg/utils"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// ApplicationUI overrides the application-chart chart
+func Volsync(m *operatorsv1.MultiClusterHub, overrides map[string]string) *unstructured.Unstructured {
+	sub := &Subscription{
+		Name:      "volsync-addon-controller",
+		Namespace: m.Namespace,
+		Overrides: map[string]interface{}{
+			"pullSecret": m.Spec.ImagePullSecret,
+			"hubconfig": map[string]interface{}{
+				"replicaCount": utils.DefaultReplicaCount(m),
+				"nodeSelector": m.Spec.NodeSelector,
+				"tolerations":  utils.GetTolerations(m),
```

Comment:
I cross-checked the overrides with the chart and it doesn't look like the chart is consuming these tolerations. They are currently hardcoded in the deployment https://github.com/stolostron/volsync-addon-controller-chart/blob/main/stable/volsync-addon-controller/templates/volsync-addon-controller-deployment.yaml#L81-L87

Reply:
@JakobGray that chart has now been updated to look similar to the others we consume